webservices/src/main/java/com/javaschool/webservices/service/impl/ShippingRabbitRPCServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.javaschool.webservices.service.impl;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.javaschool.webservices.model.PackageType;
+import com.javaschool.webservices.model.ShippingRabbitMessages;
+import com.javaschool.webservices.service.ShippingService;
+
+@Service
+public class ShippingRabbitRPCServiceImpl implements ShippingService {
```

Comment:
nitpick: This name confuses me a little, because you are saying the responsibility of this service is to Ship things, but the only method it has a the moment is related to get the package types, which does not correspond with the name.